tools/warnings_count/create_new_warnings.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# This script can be used when you want to test locally the amount of warnings produced by your changes before doing a PR.
+
+make distclean
+make -j4 setup 2> tools/warnings_count/warnings_setup_new.txt
+make -j4 all 2> tools/warnings_count/warnings_build_new.txt
```

Comment:
These scripts will only work if they're run from the parent `mm/` directory. 
A common idiom is: `DIR="$(dirname "$(readlink -f "$0")")"`, you could then `cd "$DIR/../.."` to ensure all the paths are relative to `mm/`?

Also I'd recommend adding `set -e`: we shouldn't try the `make all` step if the `make setup` step failed.

```bash
#!/bin/bash
set -e

# This script can be used when you want to test locally the amount of warnings produced by your changes before doing a PR.

DIR="$(dirname "$(readlink -f "$0")")"
cd "$DIR/../.."

make distclean
make -j4 setup 2> tools/warnings_count/warnings_setup_new.txt
make -j4 all 2> tools/warnings_count/warnings_build_new.txt
```

Reply:
Should this script be called `check_new_warnings.sh` instead? 
Should it run `tools/warnings_count/compare_warnings.py` at the end?

Reply:
I like those suggestions, I'll add them